packages/Takeover/src/components/Header/Header.styles.js

```diff
@@ -28,10 +28,32 @@ export const Header = styled.div(
 );
 
 export const Heading = styled(OriginalHeading)`
-  margin: 0 0 0 ${spacer(2)};
+  margin: 0;
+  padding: 18px 0 17px ${spacer(3)};
 `;
 
 export const CloseButton = styled.div`
   border-left: 1px solid ${tokens.border.color};
-  padding: ${spacer(1)};
+  padding: 16.5px ${spacer(2)} ${spacer(2)} 15.5px;
 `;
+
+export const HeaderRightContainer = styled.div`
+  align-items: center;
+  display: flex;
+`;
+
+export const ToolContainer = styled.div(
+  () => css`
+    padding-right: ${spacer(2)};
+    button {
```

Comment:
Would `>*` selector be better? or maybe just let the consumer to handle this part, we don't know what will be inserted in the tool container

Reply:
Should we document the `tools` api in the readme file then? Otherwise the consumer won't know we already added margins
and I think using `[data-pka-anchor="button"]` would be better if we want to do this

Reply:
yes it is already in the readme files :) I just update the description a bit so it will included required button size
![Screen Shot 2022-03-14 at 11 59 36 AM](https://user-images.githubusercontent.com/92405042/158242437-efe44692-642f-4da5-8d54-549e1fd2e09c.png)